changelog.d/10820.misc

```diff
@@ -0,0 +1 @@
+Fix a long-standing bug where an `m.room.text` event containing a null byte would cause an internal server error.
```

Comment:
```suggestion
Fix a long-standing bug where an `m.room.message` event containing a null byte would cause an internal server error.
```

I don't think m.room.text is a thing

Reply:
Ah sugar, excellent spot! `m.text` is a thing so I got confused.